src/main/java/org/sonarlint/intellij/ui/ReportTabStatusPanel.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2023 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.ui
+
+import com.intellij.openapi.project.Project
+import com.intellij.ui.components.JBPanel
+import java.awt.BorderLayout
+
+class ReportTabStatusPanel internal constructor(private val project: Project) : JBPanel<ReportTabStatusPanel>(BorderLayout()) {
+    companion object {
+        private const val HELP_TEXT: String = "This view shows simple issues found on multiple files. "
```

Comment:
I think we should also say that it's a "snapshot", so that people know the view won't be updated after the report has been created. Also that issues appear here after a specific action, it's not automatic